user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+  won’t be displayed - skip the step.
+  
+  |
+  
+  |EmailSync2|
+
+  |
+  
+- In order to **receive emails** from the external mailbox in OroCRM:
+
+ - Check the *"Enable IMAP"* box.
+
+ - Provide your IMAP credentials: host, port, and encryption type (contact your administrator for assistance).
+
+- In order to **synchronize emails sent** from OroCRM in to the external mailbox:
+
+ - Check the *"Enable SMTP"* box.
+
+ - Provide your SMTP credentials: host, port, and encryption type (contact your administrator for assistance).
+
+.. hint::
+
+    If you choose not to enable SMTP synchronization, you will still be able to send emails from OroCRM, but they won’t 
+    be synced with the email server and you will not see them in other email clients (such as Outlook or Gmail web 
+    interface). We strongly recommend to enable SMTP at all times.
+
+- In order to **finish the synchronization**:
+
+ - Provide your access credentials: your login/username (usually the email address itself) and password used to access 
+   the mailbox.
+   
+ - Click the :guilabel:`Check Connection/Retrieve Folders` button. 
+
+ - Once connection has been established , you will see the list of folders. Check the folders you want to synchronize 
+   with OroCRM.
+
+ - Save the user profile.
+  
+  |
+  
+  |EmailSyncYahoo|
+  
+  |  
+
+Syncing with Gmail
+""""""""""""""""""
+
+You may use the generic IMAP/SMTP synchronization described above for your Gmail account, however,you must allow access 
+for *"less secure apps"* in your Gmail settings first. (Please see detailed instructions 
+`here <https://support.google.com/mail/troubleshooter/1668960?hl=en&rd=1#ts=1665018%2C1665144>`_.) 
+
+To avoid this step and improve security we strongly recommend to use the dedicated functionality described below.
+This section applies to both @gmail.com customers and Google Apps customers.(Check with your email administrator if you 
+doubt).
+
+- Select the Account Type - *"Gmail"* and click the :guilabel:`Connect` button that appears below.
+
+  |
+  
+  |EmailSync3|
+  
+  |
+  
+.. note::
+
+    The option is only available only if your OroCRM instance is connected with the Google Apps account. Please check 
+    with your system administrator if you want to enable Gmail synchronization for your account.
+
+- If you haven’t used the Google account in the browser (or its cache has been cleared), the Sign-in form will appear. 
+  Log-in to the account.
+
+   |
+  
+   |EmailSyncGM1|
+
+   |
+
+ - If you are using several Gmail accounts in the browser, you will see a list of them. Select the account for which you 
+   want to enable synchronization with OroCRM. Select the account and log-in (if necessary).  
+   
+   |
+   
+   |EmailSyncGM2|
+   
+   |
+   
+ - As soon as you have selected the account, or if you are already logged in into a single Gmail account, you need to 
+   allow OroCRM to view and manage your mail, and allow it offline access to your mailbox.
```

Comment:
Edit: As soon as you are logged into a single Gmail account, you need to let OroCRM view and manage your mail, and give it offline access to your mailbox.